dev/CMakeLists.txt

```diff
@@ -109,3 +109,15 @@ target_link_libraries(${EXE_NAME}
   ${SDL_LIBRARIES}
   ${ADDITIONAL_LIBRARIES}
   ${OPENGL_LIBRARIES})
+
+if(UNIX)
+   include(GNUInstallDirs)
+   set(BINDIR ${CMAKE_INSTALL_FULL_BINDIR})
+   set(DATADIR ${CMAKE_INSTALL_FULL_DATADIR})
+else()
+   set(BINDIR .)
+   set(DATADIR .)
+endif()
+set_source_files_properties(src/platform.cpp PROPERTIES COMPILE_FLAGS -DDATADIR="\\\"${DATADIR}/lobster-data/\\\"")
```

Comment:
Why `lobster-data` instead of `lobster` ? The location already indicates it is data.

Reply:
On non-unix platforms, the installed hierarchy will contain something like:
```
lobster
lobster-data/
```

I can't name both the directory and the executable `lobster`.

Reply:
Could be just `data` I guess, but then there will be a `data/data` directory.

Reply:
We're not doing this on on non-unix platforms. Lets make sure these modifications you're making are only for platforms that desire to have data in `/usr/share` or similar. For others we're keeping existing layout.

On a side-note, I'd like to rename the `lobster` dir, since it is way too confusing in documentation and scripts. Not sure what to call it though.. any ideas?

Reply:
What about getting rid of the `lobster` directory and moving everything up one level?

Reply:
I was going to move `docs` up, but it still makes sense to have the other files in one dir?

The original idea was the `lobster` dir is everything a user of the language needs, including the exe in that dir (which is Windows centric, sorry). Not sure if I want to dump all of that in the root.

Maybe you're right though.. will have to think about it.

Reply:
Well, you got your wish.. moved everything in `lobster/*` to the root. Easier this way.